### ISSUE TITLE
ConnectRetriesInterval default not working in Maven plugin (Regression of #3499)

### ISSUE THREAD

**Issue by mihalyr** [flyway/flyway#3636]
###### Which version and edition of Flyway are you using?

Maven plugin 9.16.0

###### If this is not the latest version, can you reproduce the issue with the latest one as well? (Many bugs are fixed in newer releases and upgrading will often resolve the issue)

It is the latest

###### Which client are you using? (Command-line, Java API, Maven plugin, Gradle plugin)

Maven plugin

###### Which database are you using? (Type & version)

PostgreSQL 15.2

###### Which operating system are you using?

Linux 6.1.15-200.fc37.x86_64 (Fedora 37 Silverblue)

###### What did you do? (Please include the content causing the issue, any relevant configuration settings, the SQL statement(s) that failed (if any), and the command you ran)

```
mvn flyway:migrate
```

###### What did you expect to see?

According to the [documentation](https://documentation.red-gate.com/fd/connect-retries-interval-184127491.html) the connect retry interval should be 120, but this is not the case. There is a very visible difference if I set the interval explicitly to 120 (I can see only 3 retries) and when I leave the default (I can see my terminal scrolling from the retries).

This is most likely a regression of the bug reported in https://github.com/flyway/flyway/issues/3499 and fixed in Flyway 9.16.0.

###### What did you see instead?

Lots of repeated retries with minimum interval between them.

**Comment by wilkinsona** [comment]
I think the change in behaviour is due to [this change](https://github.com/flyway/flyway/commit/5d82f9b8a13468fd25a440a773143c00a371fb61#diff-15898f318a5fc70b6f4aefb879b5e5c0ef790996ef63fe17635c7114a12086d6) where the default values for `connectRetries` and `connectRetriesInterval` have swapped over.

Prior to that commit, the defaults were the following:

```java
model.connectRetries = 0
model.connectRetriesInterval = 120
```

They are now the following:

```
private Integer connectRetries = 120;
private Integer connectRetriesInterval = 0;  
```

I think this affects Flyway generally, not just when used through its Maven plugin.

**Comment by mihalyr** [comment]
@wilkinsona good catch! That looks definitely like a mistake as the documentation says otherwise

* [Connect Retries](https://documentation.red-gate.com/fd/connect-retries-184127492.html) should default to `0`
* [Connect Retries Interval](https://documentation.red-gate.com/fd/connect-retries-interval-184127491.html) should default to `120`

Setting this explicitly fixes it, I think I will just leave the explicit values in my config as the defaults were broken already twice in a short period.

**Comment by Barry-RG** [comment]
A PR for this is currently being processed and will be fixed in a future release.
